### PR TITLE
📄 Renamed File for Better Understanding

### PR DESCRIPTION
**Pull request by kadirnar**
@stevenliu and @yiyixu

Renamed the 'rl' file to 'run_locomotion'. This change was made to improve the clarity and readability of the codebase. The 'rl' name was ambiguous, and 'run_locomotion' provides a more clear description of the file's purpose.

Thanks 🙌

### CODE REVIEW

**Review by stevhliu — COMMENTED** [review]
I think the `rl` folder is supposed to be more general for reinforcement learning examples and "locomotion" is just an example of it. cc @patrickvonplaten who may have more insight

**Comment by pcuenca** [comment]
I think @stevhliu is right, `rl` is meant to be "reinforcement learning" :)

File: examples/reinforcement_learning/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Overview
+"# Overview
```
Comment (patrickvonplaten) [inline examples/reinforcement_learning/README.md:1]:
```suggestion
# Overview
```

**Review by stevhliu — COMMENTED** [review]
Thanks! I think the last step is to update the link [here](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) since it'll break when the folder is renamed to `reinforcement_learning`.

**Review by patrickvonplaten — COMMENTED** [review]
@stevhliu wdyt?

**Comment by stevhliu** [comment]
Ok for me as long as this [link](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) is also updated so nothing breaks :)

**Comment by kadirnar** [comment]
> Ok for me as long as this [link](https://github.com/huggingface/diffusers/blob/f6ce3236331e1981e11a1e6c790bb3e717a5ae43/examples/README.md?plain=1#L47) is also updated so nothing breaks :)

Sorry. I fixed it. Can you check?

**Pull request merged**